services/frontend/pkg/revaconfig/config.go

```diff
@@ -47,6 +47,7 @@ func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error
 		"blacklisted_files": []string{},
 		"undelete":          true,
 		"versioning":        true,
+		"tags":              true,
```

Comment:
oc10 tags are not announced via capabilities?

Reply:
Not sure, I have to check… the web implementation strictly checks for the capability…. So if we want to have it there, we need to announce it here.

Reply:
That's clear. What I wanted to say is that if oc10 also announces tags via this capability we need a different one. Because the ocis tags implementation is not compatible with the oc10 tags implementation, API wise. But I think tags management in oc10 is not even exposed via any API. So we're probably save. Still good to check it.

Reply:
True, I’ll check it tomorrow, thx for the hint 😘

Reply:
@kulmann tags are not part of the oc10 capabilities!